packages/mobile/src/verify/saga.ts

```diff
@@ -603,35 +612,34 @@ export function* completeAttestationsSaga() {
     })
   )
 
-  yield put(setMtwAddress(komenci.unverifiedMtwAddress))
-  yield put(succeed())
+  if (results.every((result) => result)) {
+    yield put(setMtwAddress(komenci.unverifiedMtwAddress))
+    yield put(succeed())
+  }
 }
 
 export function* resendMessagesSaga() {
-  Logger.error(TAG, `@resendMessagesSaga has started`)
+  Logger.debug(TAG, `@resendMessagesSaga has started`)
+
   const shouldUseKomenci = yield select(shouldUseKomenciSelector)
   const status = yield select(verificationStatusSelector)
   ValoraAnalytics.track(VerificationEvents.verification_resend_messages, {
     count: status.numAttestationsRemaining,
     feeless: shouldUseKomenci,
   })
 
-  const revealStatuses = yield select(revealStatusesSelector)
```

Comment:
I'm not sure what was the thinking behind this code, but it wasn't working. Hopefully it's more intuitive with the changes, we just set non confirmed issuers as `NotReveled` and call `revealAttestations `